### ISSUE TITLE
no texture on blocks?

### ISSUE THREAD

**Issue by wibbelo** [MazeWave/Vanilla-Raytraced#6]
I've noticed that every block is flat and glossy, and though this may be human error on my part I haven't seen anyone else talking about it. (ignore that last bit, mistakes were made)
![image](https://user-images.githubusercontent.com/80562240/111018979-e91ddb80-8381-11eb-80e2-49200dd7eca9.png)
I'm sorry if this isn't actually an active issue, I just noticed and was confused as the description mentioned textures.

**Comment by 06Games** [comment]
Hello,
What version of Minecraft are you using?
And did you download the pack from [this page](https://github.com/MazeWave/Vanilla-Raytraced/releases) ? If not, try removing the pack and reinstalling it with the correct version.


I have the impression that you cloned the repo without running the build script. If that's right, you can also just run the `make.ps1` file which will take care of generating the missing files. Once the generation process is complete you will find the `.mcpack` in `.temp/export`

**Comment by 06Games** [comment]
Closed due to inactivity

**Comment by micahlt** [comment]
@06Games Bumping this - I'm encountering the same issue.  Screenshot attached. 

![image](https://user-images.githubusercontent.com/28816297/196011702-bec8aadb-02f3-408f-b966-25d3a4f8f824.png)

Certain items also don't emit light when they should.

![image](https://user-images.githubusercontent.com/28816297/196011713-91ebb0bf-3368-498b-a818-3f47ed5acfbc.png)

**Comment by MazeWave** [comment]
This happen when Minecraft fails to parses the resources pack, try restarting the game or disable/ enable the resources pack again to fix it. This resource pack is not updated for latest version (on hold for over a year now) sorry :/

**Comment by micahlt** [comment]
No problem!  I discovered that this is a known issue (https://bugs.mojang.com/browse/MCPE-152158).  Are there any plans for updating the pack in the future, or is this project dead?

**Comment by MazeWave** [comment]
**Mostly dead**, since Mojang take **forever** to fix only criticals bugs (like crashes), I've lost nearly all interest in it. It was a lockdown project and since then life changed very quickly 😅.

I sugest you to join this discord server, it is the **biggest** comunity talking about MC RTX, lot of friends creates their own RTX resources (Vanilla and Realistic one and much more). They also devlopped a software to **deeply modify RTX settings** (like sun angle for example). They also got an updated **list of RTX related bugs and their states** (approved by Mojang, fixed...)
Hope it helps ! [Minecraft RTX Discord Invite](https://discord.gg/YyWCv6tU)

**Comment by micahlt** [comment]
Gotcha - thanks so much for your work on this!  It's a great pack!
